packages/frontend/core/src/components/affine/auth/oauth.tsx

```diff
@@ -29,11 +28,11 @@ const OAuthProviderMap: Record<
 };
 
 export function OAuth({ redirectUrl }: { redirectUrl?: string }) {
-  const serverConfig = useService(ServerConfigService).serverConfig;
+  const serverService = useService(ServerService);
   const urlService = useService(UrlService);
-  const oauth = useLiveData(serverConfig.features$.map(r => r?.oauth));
+  const oauth = useLiveData(serverService.server.features$.map(r => r?.oauth));
   const oauthProviders = useLiveData(
-    serverConfig.config$.map(r => r?.oauthProviders)
+    serverService.server.config$.map(r => r?.oauthProviders)
```

Comment:
ServerService heard weird, renamed to CloudService?

Reply:
Many places use "server", serverConfig, serverList, so server is used here for consistency.